### PR TITLE
OKTA-549236: Updating dev docs to reflect hiding sensitive data in Splunk token field

### PR DESCRIPTION
**Pull request by tomaszwrobel-okta**
Resolves: OKTA-549236

## Description:
- **What's changed?** Updated representation of Splunk HEC token field
- **Is this PR related to a Monolith release?** Yes, 2023.01.0

### Resolves:

* [OKTA-549236](https://oktainc.atlassian.net/browse/OKTA-549236)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md [hunk 1]
```diff
@@ -695,6 +695,6 @@ The Splunk Cloud Settings object specifies the configuration for the `splunk_clo
 | Property      | Description                                                  | DataType                                                       | Nullable | Unique | Readonly | MinLength | MaxLength |
 | ------------- | ------------------------------------------------------------ | -------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- |
 | host            | The domain name for your Splunk Cloud instance. Don't include `http` or `https` in the string. For example: `acme.splunkcloud.com`                                       | String                                                         | FALSE    | FALSE   | FALSE     |      17     |     116      |
-| token     | The HEC token for your Splunk Cloud HTTP Event Collector             | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |
+| token     | The HEC token for your Splunk Cloud HTTP Event Collector. Token value returned from Okta will be redacted to protect the sensitive data            | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md:698]:
```suggestion
| token     | The HEC token for your Splunk Cloud HTTP Event Collector. The token value returned from Okta is redacted to protect sensitive data.           | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |
```

File: packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md [hunk 2]
```diff
@@ -685,7 +685,7 @@ The Splunk Cloud Settings object specifies the configuration for the `splunk_clo
 
 ```json
 {
-  "host": "acme.splunkcloud.com"
+  "host": "acme.splunkcloud.com",
   "token": "11111111-1111-2222-2222-222222222222"
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md:689]:
```suggestion
  "token": "11111-redacted"
```
Is the format of the returned token `#####-redacted` or just hashed-out, like `********` ?

**Review by vanngo-okta — COMMENTED** [review]
Just a nit and a question about redacted token. Thanks!

File: packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md (hunk 2 above)
Comment (tomaszwrobel-okta) [reply packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md:688]: Upon discussion on the API shape, the token field will not be returned from the API at all

**Comment by tomaszwrobel-okta** [comment]
Hi @vanngo-okta , we decided to not return `token` field in the API payload. I introduced couple more changes to the docs, if you could please have a look again?

File: packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md [hunk 3]
```diff
@@ -238,7 +238,6 @@ curl -v -X GET \
     "status": "ACTIVE",
     "settings": {
       "host": "acme.splunkcloud.com",
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md:240]:
```suggestion
      "host": "acme.splunkcloud.com"
```

File: packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md [hunk 4]
```diff
@@ -695,6 +694,6 @@ The Splunk Cloud Settings object specifies the configuration for the `splunk_clo
 | Property      | Description                                                  | DataType                                                       | Nullable | Unique | Readonly | MinLength | MaxLength |
 | ------------- | ------------------------------------------------------------ | -------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- |
 | host            | The domain name for your Splunk Cloud instance. Don't include `http` or `https` in the string. For example: `acme.splunkcloud.com`                                       | String                                                         | FALSE    | FALSE   | FALSE     |      17     |     116      |
-| token     | The HEC token for your Splunk Cloud HTTP Event Collector             | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |
+| token     | The HEC token for your Splunk Cloud HTTP Event Collector. The token value is not returned from Okta to protect sensitive data.            | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md:697]:
```suggestion
| token     | The HEC token for your Splunk Cloud HTTP Event Collector. The token value is set during object creation, but isn't returned.            | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |
```

File: packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md (hunk 2 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md:688]: Ok, we can leave it as-is for a creation example.

File: packages/@okta/vuepress-site/docs/reference/api/schemas/index.md [hunk 5]
```diff
@@ -3380,6 +3388,7 @@ with the following keywords:
 | Property                            | Description                                                       | DataType                                                                  | Nullable | Unique | Readonly |
 | :---------------------------------- | :---------------------------------------------------------------- | :------------------------------------------------------------------------ | :------- | :----- | :------- |
 | writeOnce|Determines whether the property can be updated once it has been created|Boolean|FALSE|FALSE|TRUE|
+| writeOnly|Determines whether the property will be returned by Okta. This is to protect sensitive data|Boolean|FALSE|FALSE|TRUE|
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/schemas/index.md:3391]:
```suggestion
| writeOnly|Determines whether the property is returned by Okta to protect sensitive data|Boolean|FALSE|FALSE|TRUE|
```

**Review by vanngo-okta — COMMENTED** [review]
Looks good! A few style nits.

**Pull request merged**